.github/workflows/build.yml

```diff
@@ -62,6 +62,9 @@ jobs:
       - name: Pin tokio to 1.14 for Rust 1.45
         if: "matrix.build-net-old-tokio"
         run: cargo update -p tokio --precise "1.14.0" --verbose
+        env:
+          CARGO_NET_GIT_FETCH_WITH_CLI: "true"
+          CARGO_HTTP_MULTIPLEXING: "false"
```

Comment:
IIUC, this second line would only be necessary if we used a proxy? Sure this is required for Github Actions?

Reply:
Ah yes, looks like you're right. Let's remove it in that case since it's redundant.